flexget/plugins/sites/piratebay.py

```diff
@@ -65,20 +62,37 @@ class UrlRewritePirateBay(object):
         ]
     }
 
+    url = None
+
+    def __init__(self):
+        self.set_urls(URL)
+
+    def on_task_start(self, task, config=None):
+        if not isinstance(config, dict):
+            config = {}
+        self.set_urls(config.get('url', URL))
+
+    def set_urls(self, url):
+        if self.url != url:
+            self.url = url
```

Comment:
You should probably `rstrip` forward slashes here since you seem to make the assumption it doesn't have any later on.